gpustack/main.py

```diff
@@ -36,4 +37,5 @@ def main():
 
 
 if __name__ == "__main__":
+    freeze_support()
```

Comment:
Can we add a comment to make it clear in code.

Reply:
Updated